app/scripts/views/forms.coffee

```diff
@@ -971,6 +972,26 @@ define [
       @listenToNewFormView()
       @showNewFormViewAnimate()
 
+
+    # Request the history of the form model, the user can click on a revision and see the details
+    # preferably by simply showing the other revision as a model along side in the same views
+    fetchHistory: (formModel) ->
+
+      fielddbHelperModel = new (FieldDB.Datum)(_id: formModel.id)
+      fielddbHelperModel.fetchRevisions().then(((revisions) ->
```

Comment:
@cesine I'm trying to get this to work and I'm getting `fielddbHelperModel.fetchRevisions` as `undefined`. `fielddbHelperModel` is `Datum {_fieldDBtype: "Datum", _id: "49e123785049b1a89f49dcc666e0e6c1"}`...